client/src/components/TextMaker.tsx

```diff
@@ -24,16 +34,25 @@ const TextMaker = ({
     dangerouslySetInnerHTML: { __html: html },
   });
 };
-
+interface TMProps {
+  k: string;
+  el?: string;
+  args?: any;
+  tmf?: any;
+  style?: any;
+  className?: string;
+  template_str?: any;
+}
```

Comment:
Shouldn't these types be the same as `TextMakerProps` (with short hand names)? Any reason for the `any`s here?

Reply:
Good point, fixed